packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1524,26 +1533,22 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
           plannedInitialRoutes.add(_routeNamed<dynamic>(routeName, allowNull: true, arguments: null));
         }
       }
-      if (plannedInitialRoutes.contains(null)) {
+      if (plannedInitialRoutes.last == null) {
         assert(() {
           FlutterError.reportError(
             FlutterErrorDetails(
               exception:
                 'Could not navigate to initial route.\n'
                 'The requested route name was: "/$initialRouteName"\n'
-                'The following routes were therefore attempted:\n'
-                ' * ${plannedInitialRouteNames.join("\n * ")}\n'
```

Comment:
Seems like the variable `plannedInitialRouteNames` could be eliminated altogether then?

Reply:
We still use it below to push the routes.

Reply:
What line? Doesn't the pushing use `plannedInitialRoutes`, not `plannedInitialRouteNames`?

Reply:
I misread your previous comment, thought you were talking about `plannedInitialRoutes`.

Yes, the variable `plannedInitialRouteNames` can be eliminated. Good catch!